package.json

```diff
@@ -12,7 +12,8 @@
         "Other"
     ],
     "enabledApiProposals": [
-        "terminalShellEnv"
+        "terminalShellEnv",
+        "terminalDataWriteEvent"
```

Comment:
This will need an engine version update if the proposed API is not in stable, and a distro change.

Reply:
Yep it's old